terminus-ui/csv-entry/src/csv-entry.component.ts

```diff
@@ -653,7 +653,6 @@ export class TsCSVEntryComponent implements OnInit, OnDestroy {
   public resetTable(): void {
     this.clearAllRows();
     this.clearHeaderCells();
-    this.columnCount = DEFAULT_COLUMN_COUNT;
```

Comment:
This breaks the use case where a user pastes more columns than the default, and then clicks to reset the table. 

A test should _definitely_ have caught this, so we need to add something in.

Reply:
I tested it out in demo, it doesn’t break. The reason is that if there’s no columnCount provided, default value is used and if columnCount is provided, that value is used here for reset.
I see resetTable function is covered, so thought there’s no need to add additional tests.

Reply:
Ah I see your use case, where you can actually expand the column number on fly (via copy/paste). And reset is supposed to reset it to original set up.